### ISSUE TITLE
Blue returns green value in b2Color

### ISSUE THREAD

**Issue by kennethkufluk** [HBehrens/box2d.js#11]
My colours are giving strange results.  I've tracked it back to b2Color.js, which confuses the blue and green values here:

```
b2Color.prototype.__defineGetter__("b", function() {
                return this._g;
            });
```

I'm not sure where this error arises in the conversion process.

**Comment by HBehrens** [comment]
this is caused by a post-processing step (patch) that fixes some conversion errors. In this case it introduces another one. Will fix it in a minute.

**Comment by HBehrens** [comment]
fixed typo in b2Color.b (closed by d88915bb4997a17fd67727af02e19b4e91d8e440)

**Comment by kennethkufluk** [comment]
Thanks! :)
